x-pack/plugins/infra/docs/log_structure_helper/new_indexing_strategy_overrides.md

```diff
@@ -0,0 +1,170 @@
+# Managed asset override procedure
+
+## Terms and definitions
+
+- **asset**: An asset means an instance of a stateful Elasticsearch concept such as index, index template, data stream, ingest pipeline.
+- **managed asset**: An managed asset is an asset whose creation, update, and deletion is expected to be performed not by the user, but by a managing piece of code such as the ingest manager or a beat. Index templates are considered managed if the `managed` and `managed_by` properties of the `_meta` property are set. Data streams, indices and and ingest pipelines are considered managed if the corresponding index template is managed.
+- **new indexing strategy**: The new indexing strategy is a naming convention for assets documented in https://github.com/elastic/kibana/blob/master/x-pack/plugins/ingest_manager/dev_docs/indexing_strategy.md.
```

Comment:
There's a little note in that doc saying:

> This document is part of the original drafts for ingest management documentation in docs/ingest_manager and may be outdated. Overall documentation of Ingest Management is now maintained in the elastic/stack-docs repository.

Should we update this link to go to stack-docs?

Reply:
I couldn't find anything like this in the stack-docs repo. Do you know of a more up-to-date location?